BP/history/_BP_fix_char_history_issues.info

```diff
@@ -0,0 +1,117 @@
+	sed -i 's/lisp/lisping/g' $f
+	sed -i 's/lunatic/lunatic_1/g' $f
+	sed -i 's/martial_cleric/education_learning_2/g' $f
+	sed -i 's/mastermind_theologian/education_learning_4/g' $f
+	sed -i 's/midas_touched/education_stewardship_4/g' $f
+	sed -i 's/misguided_warrior/education_martial_1/g' $f
+	sed -i 's/monk/devoted/g' $f
+	sed -i 's/mystic/mystic_1/g' $f
+	sed -i 's/naive_appeaser/education_diplomacy_1/g' $f
+	sed -i 's/nun/devoted/g' $f
+	sed -i 's/proud/arrogant/g' $f
+	sed -i 's/quick/intellect_good_2/g' $f
+	sed -i 's/robust/physique_good_1/g' $f
+	sed -i 's/scholarly_theologian/education_learning_3/g' $f
+	sed -i 's/skilled_tactician/education_martial_3/g' $f
+	sed -i 's/slothful/lazy/g' $f
+	sed -i 's/slow/intellect_bad_1/g' $f
+	sed -i 's/socializer/reveler_2/g' $f
+	sed -i 's/stutter/stuttering/g' $f
+	sed -i 's/thrifty_clerk/education_stewardship_2/g' $f
+	sed -i 's/tough_soldier/education_martial_2/g' $f
+	sed -i 's/ugly/beauty_bad_1/g' $f
+	sed -i 's/underhanded_rogue/education_diplomacy_2/g' $f
+	sed -i '/vaishya/d' $f
+done
+
+#This was applied to regions which made the most sense; some files were reverted afterwards and redone partially by hand
+for f in ./*.txt; do 
+	echo "$f"
+	sed -i 's/trait = digambara_jain/religion = digambara/g' $f
+	sed -i 's/trait = svetambara_jain/religion = svetambara/g' $f
+	sed -i 's/trait = "svetambara_jain"/religion = svetambara/g' $f
+	sed -i 's/trait = "mahayana_buddhist"/religion = mahayana/g' $f
+	sed -i 's/trait = theravada_buddhist/religion = theravada/g' $f
+	sed -i 's/trait = vajrayana_buddhist/religion = vajrayana/g' $f
+	sed -i 's/trait = vaishnavist_hindu/religion = vaishnavism/g' $f
+	sed -i 's/trait = "vaishnavist_hindu"/religion = vaishnavism/g' $f
+	sed -i 's/trait = shaivist_hindu/religion = shaivism/g' $f
+	sed -i 's/trait = "shaivist_hindu"/religion = shaivism/g' $f
+	sed -i 's/trait = smartist_hindu/religion = smartism/g' $f
+	sed -i 's/trait = "smartist_hindu"/religion = smartism/g' $f
+	sed -i 's/trait = zurvanist_trait/religion = zurvanism/g' $f
+	sed -i 's/trait = "zurvanist_trait"/religion = zurvanism/g' $f
+	sed -i 's/trait = siyavashist/religion = siyavashism/g' $f
+	sed -i 's/trait = "siyavashist"/religion = siyavashism/g' $f
+	sed -i 's/trait = mazdeist/religion = mazdayasna/g' $f
+	sed -i 's/trait = "mazdeist"/religion = mazdayasna/g' $f
+	sed -i 's/neoplatonist/platonism/g' $f
+	sed -i 's/religion = bon/religion = old_bon/g' $f
+	sed -i 's/religion = "bon/religion = "old_bon/g' $f
+	sed -i 's/religion=bon/religion = old_bon/g' $f
+	sed -i 's/religion="bon/religion = "old_bon/g' $f
+done
+
+#Suppress sympathy for other religions
+for f in ./*.txt; do 
+	echo "$f"
+	sed -i 's/trait = sympathy/#trait = sympathy/g' $f
+	sed -i 's/trait = "sympathy/#trait = "sympathy/g' $f
+done
+
+#No Equivalents
+# AKA: Nothing in Vanilla CK3 maps well enough
+# agile, cavalry_leader, erudite, harelip
+# immortal, powerful_voice, stressed, 
```

Comment:
Immortal should not pose any problems? After all the mod does not add immortal characters.

Reply:
Caesar of dynasty 0 (WtWSMS_character_gift.txt:line 3532) is immortal. Maybe we remove the landless hall characters?

Reply:
Oh, yes, that character was a workaround to having county legion titles made into an Easter egg sort of thing, he is not meant to be ported into CK3 😉 I don't think there are other characters associated to him and his title, but if there are they can safely be removed together with him.

Reply:
I'll see if I can take time tonight to clean up dynasty = 0/immortal/landless hall issues. It shouldn't impact gameplay in any capacity while they are in the history though.